gherkin/c/CMakeLists.txt

```
@@ -70,11 +71,11 @@ target_include_directories(gherkin PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include;${CMAKE_CURRENT_SOURCE_DIR}/src>")
 
 add_executable(gherkinexe ${GHERKIN_CLI})
-target_link_libraries(gherkinexe gherkin)
+target_link_libraries(gherkinexe gherkin m)
```

```suggestion
target_link_libraries(gherkinexe gherkin PRIVATE m)
```
I don't know too much about C or CMake. But shouldn't this be linked privately so as not to expose math as part of the link interface?

---

@mpkorstanje correct but we need to link the library privately against math not the executables. I just pushed another commit that fixes this issue. Thanks!